weather-map/src/Components/Form.js

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+
+Form.propTypes = {
+    handleSubmit: PropTypes.func.isRequired,
+}
+
+function Form(props) {
+    const [inputValue, setInputValue] = useState(null);
+
+    const handleChange = ({ target }) => {
+        const { value } = target;
+        setInputValue(value);
+    }
+
+    const submitForm = () => {
+        props.handleSubmit(inputValue);
+    }
+
+    return (
+        <div className="container">
+            <div className="row d-flex justify-content-center">
+                <div className="col-sm-10">
+                    <input
+                        className="form-control"
+                        id="cityName"
+                        type="text"
+                        name="cityName"
+                        value={inputValue}
+                        onChange={handleChange}
+                        placeholder="Enter any city name"
+                    />
+                </div>
+                <div className="col-sm-2 d-flex justify-content-center">
+                    <button
+                        className="btn btn-primary ml-xl-n5 ml-md-n3"
+                        id="form-btn"
+                        onClick={() => submitForm()}
+                    >
+                        Submit
+                    </button>
```

Comment:
input va button larni form tagni ichiga olsangiz bo'lar ekan

Reply:
Ha to'g'ri aytasiz Oybek aka. Lekin manda bir muammo bo'lyaptida. Shu input va buttonni formni ichiga olsam, submitdan keyin page refresh bo'lib ketyapti